FirebaseRemoteConfig/Tests/Unit/RCNConfigDBManagerTest.m

```diff
@@ -249,6 +253,11 @@ - (void)testWriteAndLoadMetadataResult {
     XCTAssertEqual([result[RCNKeyLastApplyTime] doubleValue], now - 100);
     XCTAssertEqual([result[RCNKeyLastSetDefaultsTime] doubleValue], now - 200);
 
+    // Check that writing metadata for one namespace doesn't affect another namespace
+    NSDictionary *resultForOtherNamespace =
+        [self->_DBManager loadMetadataWithBundleIdentifier:bundleIdentifier namespace:namespace2];
```

Comment:
maybe also add a test case where you can update two namespaces at the same time.

Reply:
Added!